tests/output.rs

```diff
@@ -1113,3 +1120,67 @@ fn test_only_matching_multiline_overlapping_matches_starting_on_same_line() {
         "#,
     );
 }
+
+#[test]
+fn test_byte_offset() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @function_item' -l rust --byte-offset
+            src/helpers.rs:1:0:pub fn helper() {}
+            src/stop.rs:1:0:fn stop_it() {}
+            src/lib.rs:3:14:pub fn add(left: usize, right: usize) -> usize {
+            src/lib.rs:4:63:    left + right
+            src/lib.rs:5:80:}
+            src/lib.rs:12:139:    fn it_works() {
+            src/lib.rs:13:159:        let result = add(2, 2);
+            src/lib.rs:14:191:        assert_eq!(result, 4);
+            src/lib.rs:15:222:    }
+        "#,
+    );
+}
+
+#[test]
+fn test_byte_offset_short_option() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @function_item' -l rust -b
+            src/helpers.rs:1:0:pub fn helper() {}
+            src/stop.rs:1:0:fn stop_it() {}
+            src/lib.rs:3:14:pub fn add(left: usize, right: usize) -> usize {
+            src/lib.rs:4:63:    left + right
+            src/lib.rs:5:80:}
+            src/lib.rs:12:139:    fn it_works() {
+            src/lib.rs:13:159:        let result = add(2, 2);
+            src/lib.rs:14:191:        assert_eq!(result, 4);
+            src/lib.rs:15:222:    }
+        "#,
+    );
+}
+
+#[test]
+fn test_byte_offset_vimgrep() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @function_item' -l rust --byte-offset --vimgrep
+            src/helpers.rs:1:1:0:pub fn helper() {}
```

Comment:
This row:column:byte_offset is also what I see when running `ripgrep` with the combination of `--byte-offset` + `--vimgrep`